Documentation/operations/metrics.rst

```diff
@@ -286,146 +285,145 @@ report the eBPF memory usage directly managed by Cilium.
+``policy_max_revision``                                                                       Enabled    Highest policy revision number in the agent
+``policy_import_errors_total``                                                                Enabled    Number of times a policy import has failed
+``policy_endpoint_enforcement_status``                                                        Enabled    Number of endpoints labeled by policy enforcement status
+========================================== ================================================== ========== ========================================================
 
 Policy L7 (HTTP/Kafka)
 ~~~~~~~~~~~~~~~~~~~~~~
 
-======================================== ================================================== ========================================================
-Name                                     Labels                                             Description
-======================================== ================================================== ========================================================
-``proxy_redirects``                      ``protocol``                                       Number of redirects installed for endpoints
-``proxy_upstream_reply_seconds``                                                            Seconds waited for upstream server to reply to a request
-``proxy_datapath_update_timeout_total``                                                     Number of total datapath update timeouts due to FQDN IP updates
-``policy_l7_total``                      ``type``                                           Number of total L7 requests/responses
-======================================== ================================================== ========================================================
+======================================== ================================================== ========== ========================================================
+Name                                     Labels                                             Default    Description
+======================================== ================================================== ========== ========================================================
+``proxy_redirects``                      ``protocol``                                       Enabled    Number of redirects installed for endpoints
+``proxy_upstream_reply_seconds``                                                            Enabled    Seconds waited for upstream server to reply to a request
+``proxy_datapath_update_timeout_total``                                                     Disabled   Number of total datapath update timeouts due to FQDN IP updates
+``policy_l7_total``                      ``type``                                           Enabled    Number of total L7 requests/responses
```

Comment:
I'm not seeing these in your output either.

Reply:
Vector type metric https://github.com/cilium/cilium/blob/590388d728de5ae17f950cb8bf1cc694049522d3/pkg/metrics/metrics.go#L834